app/graphql/types/client_type.rb

```diff
@@ -0,0 +1,6 @@
+module Types
+  class ClientType < Types::BaseObject
+    field :id, ID, null: false
+    field :email, String, null: false
+  end
+end
```

Comment:
I don't know why but this feels like we can let users store some more preferences as extensions.

Reply:
Just feels sparse, but that might be good.